src/jsd/actuator.h

```diff
@@ -60,6 +60,13 @@ class Actuator : public JsdDeviceBase
   void      Reset() override;
   bool      SetOutputPosition(double position);
   bool      HasAbsoluteEncoder();
+  ActuatorStateMachineState GetActuatorStateMachineState();
+  double    GetActuatorStateCmdPosition();
+  void      SetActuatorCmdPosition(double pos);
+  void      SetActuatorCmdVelocity(double vel);
+  void      SetActuatorCmdCurrent(double cur);
+  double    UsePosCntsToEu(int32_t cnts);
+  double    UseCntsToEu(int32_t cnts);
```

Comment:
I don't like exposing these due to abstraction encapsulation. 

But it would be better to just make the `double PosCntsToEu(int32_t cnts)` public instead of make another public method that does that exact same thing as a private method.